build-tools/packages/build-infrastructure/src/types.ts

```diff
@@ -289,14 +289,17 @@ export interface IPackageManager {
 	readonly lockfileName: string;
 
 	/**
-	 * Returns an install command that can be used to install dependencies using this package manager.
+	 * Returns an array of arguments that can be used to install dependencies using this package manager.
 	 *
 	 * @param updateLockfile - If `true`, then the returned command will include flags or arguments necessary to update
 	 * the lockfile during install. If `false`, such flags or arguments should be omitted. Note that the command will
 	 * _not_ include the package manager name istself. For example, the `npm` package manager will return the string
 	 * `"install"`, not `"npm install"`.
+	 *
+	 * @example
+	 * For the pnpm package manager, calling `installCommand(true)` would return `["install", "--no-frozen-lockfile"]`.
```

Comment:
Or update if going with my rename suggestion.
```suggestion
	 * For the pnpm package manager, calling `installCommandArgs(true)` would return `["install", "--no-frozen-lockfile"]`.
```